enrolments/management/commands/load_initial_data.py

```diff
@@ -0,0 +1,112 @@
+        parser.add_argument(
+            "--verbose",
+            action="store_true",
+            help="Verbose messages",
+            default=True,
+        )
+
+    def handle(self, *args, **options):
+        num_families = options.get("families")
+        num_sessions = options.get("sessions")
+        num_classes_per_session = options.get("classes_per_session")
+        verbose = options.get("verbose")
+
+        try:
+            with transaction.atomic():
+                Enrolment.objects.all().delete()
+                Class.objects.all().delete()
+                Session.objects.all().delete()
+                Field.objects.all().delete()
+                Family.objects.all().delete()
+                Student.objects.all().delete()
+
+                registration_utils.create_test_fields()
+
+                # Create at least 1 family with guests
+                registration_utils.create_test_family_with_students(
+                    with_fields=True,
+                    num_children=fake.pyint(min_value=1, max_value=3),
+                    num_guests=1,
+                )
+
+                # Create n-1 families
+                for _ in range(num_families - 1):
+                    registration_utils.create_test_family_with_students(
+                        with_fields=True,
+                        num_children=fake.pyint(min_value=1, max_value=3),
+                        num_guests=fake.pyint(max_value=1),
+                    )
+
+                sessions = enrolment_utils.create_test_sessions(num_sessions)
+                classes = []
+
+                for session in sessions:
+                    classes.extend(
+                        enrolment_utils.create_test_classes(
+                            session=session,
+                            num_classes=num_classes_per_session,
+                        )
+                    )
+
+                num_classes = num_sessions * num_classes_per_session
+                num_families_per_class = -(-num_families // num_classes)
+                for i, class_ in enumerate(classes):
+                    enrolment_utils.create_test_enrolments(
+                        session=class_.session,
+                        enrolled_class=class_,
+                        families=Family.objects.all()[
+                            (i * num_families_per_class) : (i + 1)
+                            * num_families_per_class
+                        ],
+                    )
+
+            if verbose:
+                self.stdout.write(
+                    self.style.SUCCESS(
+                        f"Successfully created {Field.objects.all().count()} fields, "
+                        f"{Family.objects.all().count()} families, and "
+                        f"{len(Student.objects.all())} students"
+                    )
+                )
+
+        except:
+            if verbose:
+                self.stdout.write(self.style.ERROR(f"Something went wrong"))
+            raise
```

Comment:
should we have a new line here after this line?

Reply:
there is! i think github just doesn't show it - if a newline is missing it usually has a lil ⛔ symbol